pkg/controlplane/reconcilers/peer_endpoint_reconcilers.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package reconcilers Endpoint Reconcilers for the apiserver
+package reconcilers
+
+import (
+	"net"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// PeerEndpointReconciler knows how to reconcile the endpoints for the apiserver service.
+type PeerEndpointReconciler interface {
```

Comment:
we only need the lease reconciler

Reply:
Yeah this reconciler is reconciling leases now. Updated name to `PeerEndpointLeaseReconciler`